doc/sphinx/source/quickstart/installation.rst

```diff
@@ -705,18 +705,19 @@ Usually mamba is much better at solving new environments than updating older
 environments, so it is often a good idea to create a new environment if updating
 does not work.
 
-It can help mamba if you let it know what version of certain packages you want,
-for example by running
+It is possible to create a new
```

Comment:
The whole thing can be removed, as this was only relevant when we were still using conda: https://github.com/ESMValGroup/ESMValTool/blob/b8da8c03a4b67b2dd4c995df4756785155fe59a1/doc/sphinx/source/quickstart/installation.rst?plain=1#L687-L720

Reply:
687-720 will get nuked imminently, bud, cheers :beer:

Reply:
done in a8455bf81e826241fb0994b413316f65ba4c84a8

Reply:
...and a9ff3bd756b7d3fbf56746cdfa3c8297edbfaf61 :sweat_smile: